### PR TITLE
Updated Blockchain Scrapper tutorial

### PR DESCRIPTION
**Pull request by mestorlx**

### CODE REVIEW

File: documentation/tutorials/BlockchainScraper.md [hunk 1]
```diff
@@ -63,18 +68,20 @@ For your Go code, also add an entry into the docker-compose file so that your Do
...
-    options:
-      max-size: "50m"
-  secrets:
-    - api_diadata
+  btc:
+    build:
+      context: ../../../..
+    build:
+      context: $GOPATH
+      dockerfile: $GOPATH/src/github.com/diadata-org/diadata/build/Dockerfile-btc
```
Comment (frranck) [inline documentation/tutorials/BlockchainScraper.md]: 2 build sections ?

File: documentation/tutorials/BlockchainScraper.md [hunk 2]
```diff
@@ -6,52 +6,57 @@ We use Go modules for our scrapers, so that each data provider is living as an i
...
-CMD ["blockchain-scrapers"]
+
+COPY --from=build /go/bin/btc /bin/btc
+
+CMD ["btc"]
 ```
 
 After cloning a container capable of executing Go, the files from the working directory are copied into the container.
-Next, the go program located in `diadata/blockchain-scrapers/cmd` is built and installed and the finished binary is placed into a mininal distroless container.
+Next, the go program located in `iadata/internal/pkg/blockchain-scrapers/blockchains/bitcoin/scrapers/btc` is built and installed and the finished binary is placed into a mininal distroless container.
```
Comment (frranck) [inline documentation/tutorials/BlockchainScraper.md]: diadata/internal/pkg/b

File: documentation/tutorials/BlockchainScraper.md (hunk 1 above)
Comment (mestorlx) [reply documentation/tutorials/BlockchainScraper.md]: I copied that from `docker-compose.bitcoin.yml` currently that is the content on the file.
Should I update the doc to
``` Docker
    build:
      context: ../../../..
      context: $GOPATH
      dockerfile: $GOPATH/src/github.com/diadata-org/diadata/build/Dockerfile-btc
```

File: documentation/tutorials/BlockchainScraper.md (hunk 2 above)
Comment (mestorlx) [reply documentation/tutorials/BlockchainScraper.md]: fixed

File: documentation/tutorials/BlockchainScraper.md (hunk 1 above)
Comment (frranck) [reply documentation/tutorials/BlockchainScraper.md]: Ah OK

**Comment by frranck** [comment]
Thanks

**Pull request merged**